src/demos/calcite-input.html

```diff
@@ -261,7 +261,9 @@ <h1>Calcite Input</h1>
               </calcite-input>
             </calcite-label>
             <calcite-label>
-              with button with icon
+              <!-- TO-DO: fix issue with input-action button's height when button has no text -->
+              <!--   or is this not a common use case? -->
+              with no-text button and icon
```

Comment:
this is where it's setting the button's fixed height of 44px (ie., overriding the button styles from input.scss)
https://github.com/Esri/calcite-components/blob/master/src/components/calcite-button/calcite-button.scss#L527

Reply:
okay, would we remove this after the refactor of button?